### ISSUE TITLE
Filters not working for DateTime?

### ISSUE THREAD

**Issue by NirvanKS** [apstanisic/directus-dart#18]
I have the following code when trying to filter by a DateTime field.
I have also tried using the string 'now' instead of manually creating a DateTime object.

```
final announcementsList = await sdk.items('announcements').readMany(
            filters: Filters({
          'and': Filter.and([
            {
              "date_effective": Filter.lte(
                  '${DateFormat('yyyy-MM-dd HH:mm:ss').format(DateTime.now())}')
            },
            {
              "date_expired": Filter.gte(
                  '${DateFormat('yyyy-MM-dd HH:mm:ss').format(DateTime.now())}')
            }
        ])
   }));
```

Any insight would be appreciated. I would like to know if I am doing something wrong.

**Comment by apstanisic** [comment]
I found the bug. I'll publish new version when I fix failing tests because of NNBD migration.

Workaround for now is to also pass `Query`. If you don't pass query, filters are not applied
This should work:
```dart
await sdk.items('announcements').readMany(
      query: Query(), // Add this line. This is temporary workarount
      filters: Filters(
        {
          'and': Filter.and([
            {
              "created_at":
                  Filter.lte('${DateFormat('yyyy-MM-dd HH:mm:ss').format(DateTime.now())}')
            },
            {
              "created_at":
                  Filter.gte('${DateFormat('yyyy-MM-dd HH:mm:ss').format(DateTime.now())}')
            }
          ])
        },
      ));
```

**Comment by NirvanKS** [comment]
Thank you very much, this is great. I will let you know if it doesn't work.

I like the work you have done so far on this package, I hope the community grows around it.

**Comment by apstanisic** [comment]
I've published `0.6.0`, that fixes the problem. Reopen the issue if bug is not fixed.
